### PR TITLE
Cp 14

### PR DESCRIPTION
**Pull request by awan-umair**
The getAboutUsApi cp-16 and updateAboutUsApi cp-15

### CODE REVIEW

File: spring-boot-angular-15-crud-example-master/spring-boot-server/src/main/java/com/bezkoder/spring/jpa/h2/dto/AboutUsRequestDTO.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getDescription() {
+        return description;
+    }
```
Comment (Mushaf-Cyber) [inline spring-boot-angular-15-crud-example-master/spring-boot-server/src/main/java/com/bezkoder/spring/jpa/h2/dto/AboutUsRequestDTO.java:25]: same here use lombok annotation

**Review by Mushaf-Cyber — CHANGES_REQUESTED** [review]
Please resolve callouts

File: spring-boot-angular-15-crud-example-master/spring-boot-server/src/main/java/com/bezkoder/spring/jpa/h2/Entity/AboutUs.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
```
[the remark this answers is not in the document]
Comment (awan-umair) [reply spring-boot-angular-15-crud-example-master/spring-boot-server/src/main/java/com/bezkoder/spring/jpa/h2/Entity/AboutUs.java:50]: Done

File: spring-boot-angular-15-crud-example-master/spring-boot-server/src/main/java/com/bezkoder/spring/jpa/h2/dto/AboutUsRequestDTO.java (hunk 1 above)
Comment (awan-umair) [reply spring-boot-angular-15-crud-example-master/spring-boot-server/src/main/java/com/bezkoder/spring/jpa/h2/dto/AboutUsRequestDTO.java:25]: Done

**Review by Mushaf-Cyber — APPROVED** [review]
approved

**Pull request merged**